src/accelerate/components/Image.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import ContentLoader from 'react-content-loader';
+
+type Props = {
+	alt?: string,
+	className?: string,
+	src: string,
+	width?: number,
+	height?: number,
+};
+
+const loaderProps = {
+	speed: 2,
+	foregroundColor: "#f5f6f8",
+	backgroundColor: "#fff",
+};
+
+export default function Image( props: Props ) {
+	const [ isLoaded, setIsLoaded ] = useState( false );
+	return (
+		<>
+			<ContentLoader
```

Comment:
Is there a reason we don't wrap this in a conditional rather than `display: none` when it's not needed?

Reply:
🤔  not necessarily, we could do. I was thinking of having some fade in transition by layering the image over the loader but it was getting a bit fiddly

Reply:
Ahh ok, that's fine then yeah if we want a transition now / in the future.